README.md

```diff
@@ -127,6 +127,43 @@ Staged changes can be reviewed like this:
 $ nixpkgs-review wip --staged
 ```
 
+## Using nix-review in scripts
+
+After building, `nixpkgs-review` will normally start a `nix-shell` with the
+packages built, to allow for interactive testing. To use `nixpkgs-review`
+non-interactively in scripts, use the `--no-shell` command, which can allow for
+batch processing of multiple reviews or use in scripts/bots.
+
+Example testing multiple unrelated PRs:
+
+```bash
+for pr in 807{60..70}; do
+    nixpkgs-review pr --no-shell $pr && echo "PR $pr succeeded" || echo "PR $pr failed"
```

Comment:
Actually you can even do `nixpkgs-review pr 807{60..70}`

Reply:
That wouldn't get the individual follow-up command on each PR, though, would it?  The minimalist example is just to show here that you can "run a nixpkgs-review and then some cmd, based on whether it succeeded or failed for that particular review".  The `echo` is a bit of a placeholder here.